resources/lang/en/response/index.php

```diff
@@ -75,37 +81,51 @@
       ],
       'need_help' => [
         'title' => 'Does your department need extra help?',
-        'body' => 'If your department has identified a current or anticipated talent shortage in a critical area, get in touch to request some extra help. If your department has a unique need in a specialized field, we can launch a new call-out. ',
+        'body1' => 'This tool is currently available for the digital and technology community.',
+        'body2' => 'If your department has identified a current or anticipated talent shortage in a critical area, get in touch to request some extra help. If your department has a unique need in a specialized field, we can launch a new call-out. If you would would like to use this tool for another area please get in touch with your head of community or let us know. ',
         'learn_more' => [
           'text' => 'Learn More',
           'title' => 'Click here to learn more about contacting us.',
         ],
       ],
       'apply_list_title' => 'You should apply to be in the GC talent reserve if:',
```

Comment:
Nic asked me to capitalize "talent reserve" here, I'll just add it to this branch.